bot/bot.py

```diff
@@ -77,7 +77,7 @@ def _recreate(self) -> None:
 
         # Its __del__ does send a warning but it doesn't always show up for some reason.
         if self._connector and not self._connector._closed:
-            log.warning(
+            log.info(
```

Comment:
While the consequences of the situation are not that big, I believe it should remain warning because this would happen due to programmer error (i.e. not manually closing before clearing/logging in).